packages/react-components/react-nav-preview/src/components/NavGroup/useNavGroup.ts

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import {
+  getNativeElementProps,
+  mergeCallbacks,
+  useEventCallback,
+  useMergedRefs,
+  slot,
+} from '@fluentui/react-utilities';
+import { useNavContext } from '../NavContext';
+import { NavGroupProps, NavGroupState } from './NavGroup.types';
+import { SelectNavGroupEvent } from '../NavContext.types';
+
+/**
+ * Create the state required to render Tab.
+ *
+ * The returned state can be modified with hooks such as useTabStyles_unstable,
+ * before being passed to renderTab_unstable.
+ *
+ * @param props - props from this instance of Tab
+ * @param ref - reference to root HTMLElement of Tab
+ */
+export const useNavGroup = (props: NavGroupProps, ref: React.Ref<HTMLElement>): NavGroupState => {
```

Comment:
you should add _unstable suffix
```suggestion
export const useNavGroup_unstable = (props: NavGroupProps, ref: React.Ref<HTMLElement>): NavGroupState => {
```